wx/lib/colourchooser/pycolourchooser.py

```diff
@@ -363,9 +363,11 @@ def hsvToColour(self, hsv):
         # Allow values to go full range from 0 to 255
         r, g, b = colorsys.hsv_to_rgb(hsv[0], hsv[1], hsv[2])
 
-        r *= 255.0
-        g *= 255.0
-        b *= 255.0
+        round_tenths = lambda x: int(x + 0.5)
```

Comment:
Is there a reason for not using the `round` builtin function?

Reply:
Dang, looks like I forgot to commit a comment explaining this...
Python2 will return a `float` from round even if the value is integral. (eg. `26.0` rather than `26`)

Since this change is supposed to ensure that `r`, `g`, and ,`b`are of type `int`, it seemed like a lambda expression to round tenths and ensure integral type was the better option.